app/static/styles/global/_colors.scss

```diff
@@ -1,2 +1,4 @@
 // palette http://paletton.com/palette.php?uid=35B0u0kpOp2fRwykOrOsqkcyxf8
 $teal: #187B74;
```

Comment:
I'm not sure about this colour scheme.  Maybe we should talk to one of the designers :P